inst/@sym/rewrite.m

```diff
@@ -109,7 +109,7 @@
 
   F = python_cmd(cmd, sym(f), how);
 
-  % maintainer note: binop_helper would sym(how)
+  % maintainer note: op_helper would sym(how)
```

Comment:
... for example, this is not true any more: maybe just say "op_helper might sym(how)"